sampo/schemas/graph.py

```diff
@@ -354,6 +368,7 @@ def _to_adj_matrix(self) -> tuple[list[GraphNode], dok_matrix, dict[str, GraphNo
         }
         id2node: dict[str, GraphNode] = {node.id: node for node in node2ind.keys()}
         adj_mx = dok_matrix((len(node2ind), len(node2ind)), dtype=np.short)
+        weight = 0
```

Comment:
why? 'weight' is always initialized when accessed

Reply:
yes, but at the same time no
initialization of 'weight' is in the 'try' zone, but accesse to 'weight' is out of 'try'
python finds this variable, but writing it this way makes more sense